apps/condo/domains/marketplace/schema/Invoice.js

```diff
@@ -0,0 +1,166 @@
+const access = require('@condo/domains/marketplace/access/Invoice')
+const {
+    ERROR_INVALID_INVOICE_ROWS,
+    ERROR_NO_INVOICE_RECEIVERS,
+    ERROR_INVOICE_ALREADY_PAID,
+    INVOICE_STATUSES,
+    INVOICE_STATUS_DRAFT, INVOICE_STATUS_PAID,
+} = require('@condo/domains/marketplace/constants')
+
+const ERRORS = {
+    NO_RECEIVERS: {
+        code: BAD_USER_INPUT,
+        type: ERROR_NO_INVOICE_RECEIVERS,
+        message: 'Nobody sees this invoice. You must set property+unitType+unitName or accountNumber',
+        messageForUser: 'marketplace.invoice.error.noReceivers',
+    },
+    ALREADY_PAID: {
+        code: BAD_USER_INPUT,
+        type: ERROR_INVOICE_ALREADY_PAID,
+        message: 'Changing of paid invoice is forbidden',
+        messageForUser: 'marketplace.invoice.error.alreadyPaid',
+    },
+}
+
+const ajv = new Ajv()
+addFormats(ajv)
+
+const rowsFieldSchema = {
+    type: 'array',
+    items: {
+        type: 'object',
+        properties: {
+            name: { type: 'string' },
+            toPay: { type: 'string' },
+            count: { type: 'integer' },
+            vat: { type: 'string' },
+            salesTax: { type: 'string' },
+            sku: { type: 'string' },
+        },
+        required: ['name', 'toPay'],
+        additionalProperties: false,
+    },
+}
+
+const validateRowsField = getGQLErrorValidator(ajv.compile(rowsFieldSchema), ERROR_INVALID_INVOICE_ROWS)
+
+const Invoice = new GQLListSchema('Invoice', {
+    schemaDoc: 'Invoice model contains information about paid items and payer',
+    fields: {
+
+        context: {
+            schemaDoc: 'The invoice context the invoice was created for',
+            type: 'Relationship',
+            ref: 'InvoiceContext',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.PROTECT' },
+        },
+
+        number: {
+            schemaDoc: 'The invoice number within organization',
+            type: AutoIncrementInteger,
+            isRequired: true,
+            autoIncrementScopeFields: ['context.organization'],
+        },
+
+        property: {
+            schemaDoc: 'The payer\'s property',
+            type: 'Relationship',
+            ref: 'Property',
+            knexOptions: { isNotNullable: false },
+            kmigratorOptions: { null: true, on_delete: 'models.SET_NULL' },
+        },
+
+        unitType: {
+            schemaDoc: 'The payer\'s unitType',
+            type: 'Text',
+        },
+
+        unitName: {
+            schemaDoc: 'The payer\'s unitName',
+            type: 'Text',
+        },
+
+        accountNumber: {
+            schemaDoc: 'The payer\'s accountNumber within organization',
+            type: 'Text',
+        },
+
+        toPay: {
+            schemaDoc: 'How much money the payer must pay',
+            type: 'Decimal',
+            isRequired: true,
+        },
```

Comment:
You can change it to `MONEY_AMOUNT_FIELD` field template from `common` module

Reply:
done